packages/node/src/serverless-functions/serverless-handler.mts

```diff
@@ -21,40 +21,59 @@ type ServerlessFunctionSignature = (
   res: ServerResponse | VercelResponse
 ) => void;
 
-async function createServerlessServer(
-  userCode: ServerlessFunctionSignature,
-  options: ServerlessServerOptions
-) {
-  const server = createServer(async (req, res) => {
-    if (options.shouldAddHelpers) await addHelpers(req, res);
-    return userCode(req, res);
-  });
+const [NODE_MAJOR] = process.versions.node.split('.').map(v => Number(v));
+
+/* https://nextjs.org/docs/app/building-your-application/routing/router-handlers#supported-http-methods */
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Is there a plan to produce a Vercel page for supported methods?

Reply:
Yes, it will be updated in a follow-up PR 🙂